docs/secrets.md

```diff
@@ -68,6 +68,52 @@ repository.
 Workflow secrets are accessed via environment variables, in the same way
 as normal Bazel actions shown above.
 
+## Sort-lived secrets
+
+For secrets that have a short Time To Live(TTL) duration, or ephemeral secrets,
+BuildBuddy provides special headers that could be set via Bazel command line.
+
+```bash
+# For simple secrets
+--remote_exec_header=x-buildbuddy-platform.env-overrides=VAR_A=value_a,VAR_B=val_b
+
+## At execution:
+> echo $VAR_A
+value_a
+> echo $VAR_B
+val_b
+
+---
+
+# For more complex secrets
+
+## First encode the secrets using base64
+> echo -n 'VAR_C={"a": 1, "b", 2}' | base64
+> echo -n 'VAR_D=asdfa!@#C,+{}' | base64
+
+## then use the base64-encoded strings in the `env-overrides-base64` header, comma separated.
+--remote_exec_header=x-buildbuddy-platform.env-overrides-base64=VkFSX0M9eyJhIjogMSwgImIiLCAyfQ==,VkFSX0Q9YXNkZmEhQCNDLCt7fQ==
+
+## At execution:
+> echo $VAR_C
+{"a": 1, "b", 2}
+> echo $VAR_D
+asdfa!@#C,+{}
+```
+
+> Note: If 2 values are given to the same variables, the last value will be used.
+> `env-overrides` values will be applied BEFORE `env-overrides-base64` and thus,
+> the latter would take priority.
```

Comment:
I think in docusaurus you can use `:::note` and it will be rendered a bit more nicely - I guess this is rendered as a blockquote currently

```suggestion
:::note

If multiple values are given with the same variable name, the last value will be used.
If a variable is specified in both `env-overrides` and `env-overrides-base64`,
`env-overrides-base64` will take priority.

:::
```

Reply:
Oh nice! this is exactly what I was looking for.